assets/js/render-images.js

```diff
@@ -0,0 +1,13 @@
+const replaceImages = (phrase) => {
+  return phrase.replace(/(https?:\/\/\S+\.(?:png|jpg|jpeg|gif|webp))(?:$|\s)/i, "<img src=\"$1\" style=\"display: block;margin: 0 auto\">")
```

Comment:
I was thinking here...
Replacing the image URL to displaying the image we will make it difficult to copy the text and image, right?

Maybe we can implement the "COPY TEXT" button (https://github.com/ricardoruwer/caiquenator/issues/12) together, and when clicking on the copy text button, it copies the image URL 🤔

Reply:
@ferrl We just created the "COPY TEXT" button https://github.com/ricardoruwer/caiquenator/pull/19

Now you can improve this pull request!

So, the idea is: the image will be displayed as you did here, but when you click on the "COPY TEXT" button we need to copy the image URL, right?